Client.UnitTests/ModifyProcessInstanceTest.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Threading.Tasks;
+using GatewayProtocol;
+using NUnit.Framework;
+
+namespace Zeebe.Client;
+
+[TestFixture]
+public class ModifyProcessInstanceTest : BaseZeebeTest
+{
+    [Test]
+    public async Task ShouldSendRequestAsExpected()
+    {
+        // given
+        var expectedRequest = new ModifyProcessInstanceRequest
+        {
+            ActivateInstructions =
+            {
+                new ModifyProcessInstanceRequest.Types.ActivateInstruction
+                {
+                    ElementId = "test"
+                }
+            },
+            TerminateInstructions =
+            {
+                new ModifyProcessInstanceRequest.Types.TerminateInstruction()
+            }
+        };
+
+        // when
+        await ZeebeClient.NewModifyProcessInstanceCommand(processInstanceKey: 0)
+            .AddInstructionToActivate(elementId: "test", ancestorElementInstanceKey: 0)
+            .AddInstructionToTerminate(elementInstanceKey: 0)
```

Comment:
❌ please use realistic values, which you can then also assert

Reply:
I expanded this test.

https://github.com/camunda-community-hub/zeebe-client-csharp/pull/622/files#diff-0ea74d91b2de4b2794501de73beabbab0cecd442bbc7ce282a60d3f47836ba5a

It will do?